libraries/libfc/test/test_base64.cpp

```diff
@@ -1,4 +1,4 @@
-#define BOOST_TEST_MODULE base64
+#define BOOST_TEST_MODULE libfc
 #include <boost/test/included/unit_test.hpp>
```

Comment:
Did you leave `included` on purpose here?

Reply:
Hum I think the answer is yes!

Reply:
Yeah I thought that was a little sneaky how "the magic include" was simply buried in one of the test files like this. It might be clearer to have a discrete `main.cpp` with simply
```c++
#define BOOST_TEST_MODULE libfc
#include <boost/test/included/unit_test.hpp>
```

Reply:
Yes on purpose. I agree that is too sneaky, so added a `main.cpp` as Matt suggested.